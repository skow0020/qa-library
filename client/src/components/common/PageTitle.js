import PropTypes from 'prop-types';
import React from 'react';
import Typography from '@material-ui/core/Typography';

export default function PageTitle({ title }) {
  return (
      <Typography gutterBottom className="page-title" variant="h3">
        {title}
      </Typography>
  );
};

PageTitle.propTypes = {
  title: PropTypes.string.isRequired
};