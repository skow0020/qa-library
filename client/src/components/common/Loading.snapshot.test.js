import PageTitle from './PageTitle';
import React from 'react';
import renderer from 'react-test-renderer';

describe('PageTitle Snapshot Tests', () => {
  test('Link changes the class when hovered', () => {
    const component = renderer.create(
      <PageTitle></PageTitle>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});