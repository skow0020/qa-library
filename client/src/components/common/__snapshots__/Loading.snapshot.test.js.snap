// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Snapshot Tests Loading snapshot 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  style={
    Object {
      "minHeight": "60vh",
    }
  }
>
  <div
    className="makeStyles-root-1"
  >
    <div
      className="MuiCircularProgress-root makeStyles-circular-2 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      id="loading"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </div>
  </div>
</div>
`;
